hm_pyhelper/hardware_definitions.py

```diff
@@ -357,6 +357,29 @@ def is_raspberry_pi():
         'CONTAINS_IC_IDS': []
         },
 
+    # Smart Mimic / Mimiq Finestra
+    'COMP-FINESTRA': {
+        'FRIENDLY': 'Finestra Miner',
+        'SPIBUS': 'spidev0.0',
+        'KEY_STORAGE_BUS': '/dev/i2c-1',
+        'RESET': 17,
+        'MAC': 'wlan0',
+        'STATUS_LED': {
+            'TYPE': 'RGB',
```

Comment:
Nice

Reply:
I'll add some docs and bump version number then mark as ready for review 👍